src/backend/gpopt/translate/CTranslatorScalarToDXL.cpp

```diff
@@ -2172,7 +2162,8 @@ CTranslatorScalarToDXL::TranslateArrayRefToDXL(
 
 CDXLNode *
 CTranslatorScalarToDXL::TranslateSortGroupClauseToDXL(
-	const Expr *expr, const CMappingVarColId *var_colid_mapping)
+	const Expr *expr,
+	const CMappingVarColId *var_colid_mapping __attribute__((unused)))
```

Comment:
Can we delete this argument and may be some others instead of making compiler silent?

Reply:
I'd like to avoid any significant code changes, including altering of functions/methods arguments. Also, the approach with `unused` is already widely used in ORCA. There is an existing macro `GPOS_UNUSED` for this purpose. I've reworked the patch to use it.